### PR TITLE
[MM-39931] - Tutorial points: arrow doesn't get proper theme color

### PR DESCRIPTION
**Pull request by AGMETEOR**
#### Summary
Updates css files to theme tutorial point arrows properly

#### Ticket Link
[MM-3993](https://mattermost.atlassian.net/browse/MM-39931)

#### Release Note

```release-note
NONE

```

### CODE REVIEW

File: sass/components/_tutorial.scss [hunk 1]
```diff
@@ -78,7 +78,7 @@ $tip-index: 1000;
         .arrow {
             top: 34px;
             left: -10px;
-            border-right-color: $white;
```
Comment (pablovelezvidal) [inline sass/components/_tutorial.scss:81]: Thanks @AGMETEOR , This solves the problem. However, I think it is worth checking if it is only this `$white ` variable or if all the other included variables in this file are not working.

Comment (neallred) [reply sass/components/_tutorial.scss:81]: Looking over this file quickly, seems like there are other instances of `$white` that should be replaced. I think the instances of `$black` are all for shadows and can stay as they are.

Comment (AGMETEOR) [reply sass/components/_tutorial.scss:81]: @neallred  I only replaced those instances where I could see the color of the arrow being white on a dark theme. Is it safe to replace all those other `$white` variables with the `--center-channel-bg`

Comment (neallred) [reply sass/components/_tutorial.scss:81]: I haven't checked. If you checked the various tips on dark theme and they look good with the changes you have now, I'd say keep it as you have it.

**Review by neallred — APPROVED** [review]

**Comment by AGMETEOR** [comment]
@stevemudie gentle ping on this

**Review by pablovelezvidal — APPROVED** [review]

**Review by stevemudie — COMMENTED** [review]
@AGMETEOR The arrow still appears white with either of the dark themes applied.

Screen Shot 2022-01-06 at 7 30 33 AM

**Comment by AGMETEOR** [comment]
> @AGMETEOR The arrow still appears white with either of the dark themes applied.
> 
> Screen Shot 2022-01-06 at 7 30 33 AM

Hey @neallred . Is there some setting I need to touch to get the `Paybooks` tutorial tip showing locally?

**Comment by AGMETEOR** [comment]
> https://mattermost-webapp-pr-9603.test.mattermost.cloud

@stevemudie fixed

**Review by stevemudie — APPROVED** [review]
Looks good @AGMETEOR!

**Pull request merged**